prow/clonerefs/parse.go

```diff
@@ -90,35 +90,33 @@ func ParseRefs(value string) (*kube.Refs, error) {
 	return gitRef, nil
 }
 
-// ClonePathAlias provides path overrides for a given repo
-type ClonePathAlias struct {
+// OrgRepoOverride provides an override for a given repo
+type OrgRepoOverride struct {
```

Comment:
IMO clonerefs should be coupled to git, not github.

We should be talking about path prefixes, not github-specific abstractions like org and repo

Reply:
> IMO clonerefs should be coupled to git, not github.

A little too late for that, and I'm not super interested in a breaking change that I will need to chase. But we can make a couple iterative changes if you'd like. What should we default the clone URI to?

> We should be talking about path prefixes, not github-specific abstractions like org and repo

Is this not specifically exposing the clone URI prefix to the user, and not touching the org or repo?

Reply:
Agreed... looking deeper the org/repo thing is baked into [kube.GitRefs](https://github.com/stevekuznetsov/test-infra/blob/793d9a036d4c1807e85f0c957093a6d3e2e6ec5e/prow/kube/prowjob.go#L246) so a lot would have to change. Cost/benefit here isn't worth it.